pom.xml

```diff
@@ -76,7 +76,9 @@
                 <configuration>
                     <serverId>sonatype-nexus-staging</serverId>
                     <nexusUrl>https://oss.sonatype.org/</nexusUrl>
-                    <autoReleaseAfterClose>true</autoReleaseAfterClose>
+                    <autoReleaseAfterClose>false</autoReleaseAfterClose>
+                    <keepStagingRepositoryOnFailure>true</keepStagingRepositoryOnFailure>
+                    <keepStagingRepositoryOnCloseRuleFailure>true</keepStagingRepositoryOnCloseRuleFailure>
```

Comment:
Without it, I had the `mvn deploy` fail for me several times with cryptic message:
```
[ERROR] Rule failure while trying to close staging repository with ID "comsumologic-1305".
[ERROR] 
[ERROR] Nexus Staging Rules Failure Report
[ERROR] ==================================
[ERROR] 
[ERROR] 
[ERROR] Cleaning up local stage directory after a Rule failure during close of staging repositories: []
[ERROR]  * Deleting context 430d4a26915a39.properties
[ERROR] Cleaning up remote stage repositories after a Rule failure during close of staging repositories: []
[ERROR]  * Dropping failed staging repository with ID "comsumologic-1305" (Rule failure during close of staging repositories: []).
```